qa/integration-tests/src/test/java/io/camunda/zeebe/it/client/command/BroadcastSignalTest.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+
+package io.camunda.zeebe.it.client.command;
+
+import static io.camunda.zeebe.test.util.record.RecordingExporter.signalRecords;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.camunda.zeebe.broker.test.EmbeddedBrokerRule;
+import io.camunda.zeebe.it.util.GrpcClientRule;
+import io.camunda.zeebe.model.bpmn.Bpmn;
+import io.camunda.zeebe.protocol.record.Assertions;
+import io.camunda.zeebe.protocol.record.Record;
+import io.camunda.zeebe.protocol.record.intent.SignalIntent;
+import io.camunda.zeebe.protocol.record.value.SignalRecordValue;
+import io.camunda.zeebe.test.util.BrokerClassRuleHelper;
+import java.util.Map;
+import org.junit.Before;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.RuleChain;
+
+public class BroadcastSignalTest {
```

Comment:
👍 Awesome work

❌ Please add a test case that shows that a broadcasted signal can start a new process instance (using signal start event)

Reply:
Done.

Hi @korthout. Please check this out. :bow:

BTW. I'll squash the commits when you've checked. ❤️